draft-ietf-webtrans-http3.md

```diff
@@ -82,9 +82,9 @@ WebTransport servers in general are identified by a pair of authority value and
 path value (defined in {{!RFC3986}} Sections 3.2 and 3.3 correspondingly).
 
 When an HTTP/3 connection is established, both the client and server have to
-send a SETTINGS_ENABLE_WEBTRANSPORT setting in order to indicate that they both
-support WebTransport over HTTP/3.  This process also negotiates the use of
-additional HTTP/3 extensions.
+send a SETTINGS_WEBTRANSPORT_MAX_SESSIONS setting in order to indicate that
+they both support WebTransport over HTTP/3.  This process also negotiates the
```

Comment:
Unrelated to this PR, but `This process also negotiates the use of additional HTTP/3 extensions.` is no longer true now that we require negotiating each extension separately. Can we just remove that sentence?

Reply:
I believe Lucas added this in the PR related to WEBTRANSPORT_STREAM, so we may want to clarify it instead of removing.